requirements.txt

```diff
@@ -1 +1,6 @@
 requests==2.32.3
+
+urllib3~=2.3.0
+aiohttp~=3.11.11
+pytest~=8.3.4
+python-dotenv~=1.0.1
```

Comment:
Please leave only the production requirements in this file. Development/testing requirements are currently only tracked in the test github action but I'd be down for a `requirements.dev.txt` file.